osu.Game/Audio/HitSampleInfo.cs

```diff
@@ -50,9 +50,9 @@ public virtual IEnumerable<string> LookupNames
             get
             {
                 if (!string.IsNullOrEmpty(Suffix))
-                    yield return $"{Bank}-{Name}{Suffix}";
+                    yield return $"Gameplay/{Bank}-{Name}{Suffix}";
 
-                yield return $"{Bank}-{Name}";
+                yield return $"Gameplay/{Bank}-{Name}";
```

Comment:
I believe moving these prefixes locally will break every single legacy skin (including beatmap skins) with custom gameplay hitsounds, unfortunately.

Reply:
legacy skins already do local manipulation to remove all prefixes

Reply:
Should be resolved (via https://github.com/ppy/osu/pull/10619).